src/inference/src/system_conf.cpp

```diff
@@ -274,6 +276,7 @@ std::vector<std::vector<int>> reserve_available_cpus(const std::vector<std::vect
 }
 
 void set_cpu_used(const std::vector<int>& cpu_ids, const int used) {
```

Comment:
@wangleis why do we have global state for CPU? 
we have not agreed on arch forum about these functions like resverse cpu cores

Reply:
@ilya-lavrenov We have global state of CPU for sharing CPU resource in multiple models situation or multiple plugins situations. This is part of multiple threading improvement agreed in arch forum. 

For 2023.0, we will support multiple plugins situation. Would you like to skip multiple models situation as well and totally not use `set_cpu_used()`?

Reply:
I don't remember it